mozilla-central/setup

```diff
@@ -10,6 +10,9 @@ pushd $INDEX_ROOT
 if [ -d "gecko-dev" ]
 then
     echo "Found pre-existing gecko-dev folder; skipping re-download."
+    pushd $INDEX_ROOT/gecko-dev
+    git checkout master
```

Comment:
Why is this needed?

Reply:
(And why don't we need to `fetch` / `pull`?

Reply:
Could you add some of this explanation in a comment here?

Reply:
Done